packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -896,14 +898,56 @@ const Conversation = WebexPlugin.extend({
         async: false
       });
     }
+    /**
+   * helper function to cloneDeepWith for copying instance function
+   * @param {T} value
+   * @returns {null}
```

Comment:
This function can return an object as well as a null, it needs to be defined here.

Reply:
Okay, will update here.